src/cli.rs

```diff
@@ -83,6 +83,13 @@ Defaults to \"format\" if the NO_COLOR env is set and to \"none\" if stdout is n
     )]
     pub pretty: Option<Pretty>,
 
+    // Set formatting options, such as the amount of spaces per JSON indentation
+    // 
+    // This only works when pretty is set to all or format
+    // Example: --format-options=json.indent:2,json.record.separator:' EOF'
+    #[clap(long, value_name = "FORMAT_OPTIONS")]
+    pub format_options: Option<String>,
```

Comment:
Once you implement `FromStr` trait for `FormatOptions`, you can pass it to clap

```suggestion
    pub format_options: Option<FormatOptions>,
```